### ISSUE TITLE
False positive RCE (932380)

### ISSUE THREAD

**Issue by Simbiat** [coreruleset/coreruleset#3964]
### Description

I am running FrankenPHP (Caddy) in Docker ([compose file](https://github.com/Simbiat/simbiat.ru/blob/master/docker-compose.yml)) with these Coraza [settings](https://github.com/Simbiat/simbiat.ru/blob/c5526d60652c84b896874bb65596d0474e76d27f/config/frankenphp/caddy.json5#L129). When I send a text like:
```html
<p>time for him</p>
```
In a request like
```bash
curl 'https://www.simbiat.dev/api/fftracker/characters' \
  -H 'accept: */*' \
  -H 'accept-language: en-US,en;q=0.9,ru;q=0.8,fi;q=0.7' \
  -H 'content-type: multipart/form-data; boundary=----WebKitFormBoundaryBFK7eJVVI8mtxPlc' \
  -H 'cookie: __Host-sess_wwwsimbiatdev=b3kfngm9m4nqvirn1rfjbufmmepobe9mhacidvv04rt5afb3ju3ru7oitoku76f1pqfo1q6p4hjq3gvilpt7q6oqesgkk0qukk21el24dmrl0co0fo7tpb5sq7qbg48o; rememberme_www.simbiat.dev=%7B%22cookieid%22%3A%22lvnSyrK96u35lIFJ%2BqMtJZwGIBuWXRM4ZDDUVfhNmjMP8wD2oUwT%5C%2FRhl3JTKPCeWaWqzPPYLbIZxsIj9Fi3cJcgKDH6xC4yWB0tNdayibD0dVB2Wy1yIGAMKvwNyMSOWzjZyz%5C%2FBdqYe8YbPNW6uUgP2dXN2Hqf7UW1TciYuBZIXB4pCQ6hOz2auhme5EKPJwY0ptNlhXyEwav4qi%22%2C%22pass%22%3A%22D%2BoEnlGCVyXWdil%2BHqWKPuVW2MRKo52petUgdc1IZ0bKQ1Ndx%5C%2FoG8%2BGCOEDqRzh8kTk8MLF2ggCUX8hldMOyg3cyKUf5kgV8oAu3JrTxZkWDyQ5wIjgHCIAD1n4Ahn9EPz56LgyKY13aW3%5C%2FOmsBdDGGvoYI1Ml0v01Lg2h92hezKRNxyvauPietoNTuArNfSfihPQqLq4RezAN2CqGrkfiBbd2AIY9Fcocp7xnF%2Bjvb6RBtkJAP2U6T797EyvrYiT93wn2oNgN%2Bn4I%2BOtsiei%2BYR6S3%5C%2FGWzvZPof2i6D52aWLnU%2BhATPaOzOUgx6%5C%2FijZMrlBFdnDvYxo3Gix1i4Hv5CIzfiZ2nSvzgLMFSYBP5lR0WBAfjHmpR%5C%2FYaRI%3D%22%7D' \
  -H 'dnt: 1' \
  -H 'origin: chrome-extension://eipdnjedkpcnlmmdfdkgfpljanehloah' \
  -H 'priority: u=1, i' \
  -H 'sec-ch-ua: "Microsoft Edge";v="131", "Chromium";v="131", "Not_A Brand";v="24"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "Windows"' \
  -H 'sec-fetch-dest: empty' \
  -H 'sec-fetch-mode: cors' \
  -H 'sec-fetch-site: none' \
  -H 'user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36 Edg/131.0.0.0' \
  --data-raw $'------WebKitFormBoundaryBFK7eJVVI8mtxPlc\r\nContent-Disposition: form-data; name="test"\r\n\r\n<p>time for him</p>\r\n------WebKitFormBoundaryBFK7eJVVI8mtxPlc--\r\n' ;
curl 'https://www.google-analytics.com/mp/collect?measurement_id=G-3Y4R2TW5NF&api_secret=vufVkFGNTi2yFqle2W32fQ' \
  -H 'sec-ch-ua-platform: "Windows"' \
  -H 'Referer;' \
  -H 'User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36 Edg/131.0.0.0' \
  -H 'sec-ch-ua: "Microsoft Edge";v="131", "Chromium";v="131", "Not_A Brand";v="24"' \
  -H 'DNT: 1' \
  -H 'Content-Type: text/plain;charset=UTF-8' \
  -H 'sec-ch-ua-mobile: ?0' \
  --data-raw '{"client_id":"d542e03b-5b4b-437e-8d35-33a4f75b756f","events":[{"name":"send","params":{"session_id":"1735277047680","engagement_time_msec":100,"event_category":"HttpClient","event_label":""}}]}'
```
I get error like
```json
{"level":"error","ts":1735277374.5424275,"logger":"http.handlers.waf","msg":"[client \"87.92.5.57\"] Coraza: Access denied (phase 2). Remote Command Execution: Windows Command Injection [file \"@owasp_crs/REQUEST-932-APPLICATION-ATTACK-RCE.conf\"] [line \"3487\"] [id \"932380\"] [rev \"\"] [msg \"Remote Command Execution: Windows Command Injection\"] [data \"Matched Data: time for found within ARGS:test: <p>time for him</p>\"] [severity \"critical\"] [ver \"OWASP_CRS/4.0.0-rc1\"] [maturity \"0\"] [accuracy \"0\"] [tag \"application-multi\"] [tag \"language-shell\"] [tag \"platform-windows\"] [tag \"attack-rce\"] [tag \"paranoia-level/1\"] [tag \"OWASP_CRS\"] [tag \"capec/1000/152/248/88\"] [tag \"PCI/6.5.2\"] [hostname \"\"] [uri \"/api/fftracker/characters\"] [unique_id \"WCseTcPgePsxydQz\"]\n"}
```
Unfortunately, similar to #3962, not replicable on Sandbox. Let me know if debug output is required.

### Confirmation

[x] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by theseion** [comment]
This had already been fixed in 02edc46bf9593c205e0474d4ce080a9290a4ffad. The version you're using is over 11 months old.

**Comment by Simbiat** [comment]
Can confirm. I forced to use Caddy's module from `@main` which forced to use latest ruleset and this is no longer a problem.
